components/Bookmarks/BookmarkMiniFeed.js

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useRef } from 'react'
+import { compose, graphql } from 'react-apollo'
+import { css } from 'glamor'
+import { colors, fontStyles, mediaQueries, Loader } from '@project-r/styleguide'
+
+import Bookmark from '../ActionBar/Bookmark'
+import UserProgress from '../ActionBar/UserProgress'
+import withT from '../../lib/withT'
+import { withMembership } from '../Auth/checkRoles'
+import Link from '../Link/Path'
+
+import { getBookmarkedDocuments } from './queries'
+
+const BookmarkMiniFeed = ({ t, data, ...props }) => {
+  return (
+    <Loader
+      loading={data.loading}
+      error={data.error}
+      render={() => {
+        const nodes = data.me.collection.items.nodes
+        return (
+          <div {...styles.tilesContainer} {...props}>
+            {nodes.slice(0, 8).map(node => {
+              const { userProgress, userBookmark } = node.document
+              const {
+                estimatedReadingMinutes,
+                description,
+                path
+              } = node.document.meta
+              return (
+                <div {...styles.tile} key={node.id}>
+                  <Link path={path} passHref>
+                    <div
```

Comment:
this should become an a tag (otherwise `passHref` make no sense and is not cmd clickable)